frontend/online-shop/src/styles/ProductDetailStyle.tsx

```diff
@@ -0,0 +1,17 @@
+import { makeStyles, createStyles, Theme } from "@material-ui/core";
+
+export const productDetailStyle = makeStyles((theme: Theme) =>
+  createStyles({
+    headingStyle: {
+      marginTop: "50px",
+      marginLeft: "40px",
+      fontFamily: theme.typography.fontFamily,
+      display: "inline-block",
+    },
+    buttonStyle: {
+      float: "right",
```

Comment:
A nice info here, in the recent years `FlexBoxes` have grown in popularity and they can replace the floating and also achieve much more. Here is a nice presentation: https://css-tricks.com/snippets/css/a-guide-to-flexbox/